src/foundry/client-esm/helpers/_types.d.mts

```diff
@@ -1,5 +1,50 @@
-export interface CompendiumArtInfo {}
+import type { PrototypeToken } from "../data/_module.d.mts";
 
+export interface CompendiumArtInfo {
+  /**
+   * The path to the Actor's portrait image.
+   */
+  actor?: string | undefined;
+
+  /**
+   * The path to the token image, or an object to merge into the Actor's prototype token.
+   */
+  token?: string | PrototypeToken | undefined;
```

Comment:
Seems like this should be the inferred assignment type of the Prototype Token

Reply:
So my understanding of the whole assignment type hierarchy is admittedly new and fuzzy, but I'm not sure what you mean. You'd want something like `fields.SchemaField.InnerAssignemnentType<PrototypeToken.Schema>>` ?

Reply:
Yeah, and then write a test cover each possible option here.